src/style.css

```diff
@@ -1,3 +1,98 @@
+/* stylelint-disable string-quotes */
+@font-face {
+  font-family: 'Roboto';
+  src: url('fonts/Roboto-Medium.ttf') format('woff2');
+  font-weight: 500;
+  font-style: normal;
+}
+
 body {
   margin: 0;
+  padding: 0;
+  font-family: 'Roboto', sans-serif;
+}
+
+.header {
+  display: flex;
+  height: 60px;
+  background-color: #fff;
+  box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.05);
+  padding: 0 50px;
+  width: 100%;
+  box-sizing: border-box;
+}
+
+.logo-container {
+  display: flex;
+  flex-shrink: 3;
+  flex-grow: 1;
+}
+
+.navigation-block {
+  flex-shrink: 0;
+  flex-grow: 3;
+  display: flex;
+  justify-content: flex-end;
+}
+
+.logo-container a {
+  height: 40px;
+  width: 40px;
+  align-self: center;
+}
+
+nav > .menu-nav {
+  display: flex;
+  margin: 0;
+  padding: 0;
+  list-style: none;
+  height: 60px;
+  font-size: 12px;
+}
+
+.menu-nav li {
+  text-transform: uppercase;
+  display: flex;
+  align-items: center;
+  height: 100%;
+  position: relative;
+}
+
+.menu-nav li:nth-child(n+2) {
+  margin-left: 20px;
+}
+
+.menu-nav li > a {
+  height: 100%;
+  line-height: 60px;
+  white-space: nowrap;
+  color: #000;
+  font-family: 'Roboto', sans-serif;
+  font-size: 12px;
+  font-style: normal;
+  font-weight: 500;
+  text-transform: uppercase;
+}
+
+/* .menu-nav li > a[data-qa="hover"] {
+  color: #00acdc;
+} */
+
+.menu-nav li > a:hover {
+  color: #00acdc;
+  cursor: pointer;
+}
+
+.menu-nav li a.is-active {
+  color: #00acdc;
+}
+
+.menu-nav li a.is-active::after {
+  width: 100%;
+  border: 2px solid #00acdc;
+  position: absolute;
+  bottom: 0;
+  left: 0;
+  content: "";
+  border-radius: 8px;
```

Comment:
Let's try to organize your CSS
I recommend you take a look at these styles code and use it in the feature
[9elements](https://9elements.com/css-rule-order) or [htmlAcademy](https://codeguide.academy/html-css.html#css-order)